haskell/src/Monocle/Api/Server.hs

```diff
@@ -287,7 +287,12 @@ crawlerAddDoc request = do
       pure $ CrawlerPB.AddDocResponse Nothing
     addProjects crawler organizationName projects = do
       monocleLogEvent $ AddingProject (getWorkerName crawler) organizationName (length projects)
-      I.initCrawlerEntities (Project <$> projectNames projects) crawler
+      let names = projectNames projects
+          -- TODO(fbo) do not crawl github issues by default for an organization.
```

Comment:
It seems like this is already the case, shouldn't the comment be: `TODO: figure out a strategy to crawl the issues` ?

Reply:
Right, I've updated the TODO comment